internal/kafka/source_result.go

```diff
@@ -24,6 +24,8 @@ type SourceResult struct {
 	Err error `json:"error"`
 
 	Identity identity.XRHID `json:"-"`
+
+	MissingPermissions []string `json:"missing_permissions"`
 }
```

Comment:
Sources do not currently support that, so I would use `-` json tag for now. Only into logging for now.